### PR TITLE
Create ccset.js

### PR DESCRIPTION
**Pull request by ashyudhajit**
Custom color lol...by ~haji

### CODE REVIEW

**Comment by panpawn** [comment]
Our CSS is on GitHub, this doesn't sync local changes with that.

File: ccset.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+var fs = require('fs');
+var selectors;
+ 
+function writeIconCSS() {
+        fs.appendFile('config/custom.css', selectors);
+}
+ 
+exports.commands = {
+        ccset: function (target, room, user) {
+		if (user.userid !== 'ashyudhajit') return this.errorReply('/ccset - PM Ashyudhajit to set your custom color.');
```
Comment (panpawn) [inline ccset.js]: lolwhat

**Comment by panpawn** [comment]
I'm closing this because I don't think you'll be able to make the changes that this would need...

Furthermore, Gold handles custom colors already by giving them to users on our custom client, so this will complicate things for users who already have them on the client.  If I ever change my mind on this, then I'll re-open this.

**Comment by SilverTactic** [comment]
@panpawn I actually don't think this is a bad idea. You can set custom chat colors using custom css (that's what Sora does). The concept isn't a bad one, but the way it's coded looks kinda incorrect

**Comment by SilverTactic** [comment]
You could always merge local edits to custom.css later too.
Also, removing username colors could be hard. You'd probably need to add something in that css file as a comment to try and find the right user's color

**Comment by panpawn** [comment]
If this kind of system did ever get on the server, we would need it to be able to add / remove anyone's just with simple adding / removing commands.  Each time the CSS file is changed, I would prefer it to push those changes to GitHub.  

If that's possible, then I'll consider looking into this method.  (I know it's possible, it's more of a question of "if it's worth it or not.").

Next project :^)?

**Pull request closed without merging**